lib/Bio/KBase/ObjectAPI/KBaseFBA/FBAModel.pm

```diff
@@ -2411,7 +2411,7 @@ sub translate_to_localrefs {
 		my $array = [split(/_/,$reactions->[$i]->id())];
 	 	my $comp = pop(@{$array});
 	 	$comp =~ s/\d+//;
-		if ($reactions->[$i]->reaction_ref() =~ m/\/([^\/]+)$/) {
+		if ($reactions->[$i]->reaction_ref() =~ m/\/([^\/]+?)(_[^\/])?$/) {
```

Comment:
@samseaver shouldn't this be: m/\/([^\/]+)(_[^\/])?$ (no first question mark because we want one or more compound id letters)